wordpress-mobile/gutenberg-mobile
# Improve unsupported block message for reusable block

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/32618

To test:
Check the testing instructions from the `gutenberg` PR: https://github.com/WordPress/gutenberg/pull/32618

PR submission checklist:
